demos/rag_agentic/notebooks/Level4_agentic_and_mcp.ipynb

```diff
@@ -546,31 +572,55 @@
    "source": [
     "# Here is a system prompt we have come up which works well for this query\n",
     "sys_prompt3= \"\"\"You are a helpful AI assistant with access to the tools listed next. When a tool is required to answer the user's query, respond with `<tool_call>` followed by a JSON object of the tool used. For example: `<tool_call> {\"name\":\"function_name\",\"arguments\":{\"arg1\":\"value\"}} </tool_call>`:The user will respond with the output of the tool execution response so you can continue with the rest of the initial user prompt (continue).\n",
-    "If a tool does not exist in the provided list of tools, notify the user that you do not have the ability to fulfill the request. \"\"\""
+    "If a tool does not exist in the provided list of tools, notify the user that you do not have the ability to fulfill the request. \"\"\"\n",
+    "sys_prompt4=\"\"\"You are a helpful assistant. You have access to a number of tools.\n",
```

Comment:
She these have relevant names like they do in the script? 
`llama_prompt`, `granite_prompt` ?